test/lib/TestTickTree.sol

```diff
@@ -45,9 +46,9 @@ contract TestTickTree is MangroveTest {
   // Creates a snapshot of the Mangrove tick tree
   function snapshotMgvTickTree() public {
     local = reader.local(olKey);
-    Field level2 = mgv.level2(olKey);
+    Field _level2 = mgv.level2(olKey);
```

Comment:
I think this should be written to `level2` directly, otherwise level2 isn't included in the snapshot (which could explain why the tests fail):

```
level2 = mgv.level2(olKey);
```

The remaining changes in this function can be removed.

Reply:
I pushed a fix and reenabled the tests: 50bfeb09f5416030e8f62faf33e8b881151d2005